### PR TITLE
https://issues.redhat.com/browse/ACM-8900 Gatekeeper updates

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8900

### CODE REVIEW

File: governance/create_gatekeeper.adoc [hunk 1]
```diff
@@ -99,7 +101,9 @@ spec:
     other-annotation: "another test"
 ----
 +
-You can use the following values to manage your webhook operations. Use the following values for the `operations` parameter, `"CREATE"`, `"UPDATE"`, `"CONNECT"`, and `"DELETE"`.
+<1> Enable the `auditEventsInvolvedNamespace` to manage which namespace audit event to create.
```
Comment (yiraeChristineKim) [inline governance/create_gatekeeper.adoc]: This is better something like this 
```
To add --audit-events-involved-namespace arg, set to true
```

File: governance/create_gatekeeper.adoc [hunk 2]
```diff
@@ -99,7 +101,9 @@ spec:
...
+<2> Enable the `admissionEventsInvolvedNamespace` to manage which namespace admission event to create.
```
Comment (yiraeChristineKim) [inline governance/create_gatekeeper.adoc]: This is better something like this 
To add --admission-events-involved-namespace arg, set to true

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -62,6 +62,8 @@ See link:../observability/observe_environments_intro.adoc#observing-environments
 
 * The Gatekeeper operator exposes a setting in the custom resource definition within the `auditFromCache` audit, which is disabled by default. If you enable `auditFromCache`, then you need to set `config.gatekeeper.sh` for the sync details. See link:../governance/create_gatekeeper.adoc#managing-gatekeeper-operator-policies[Managing Gatekeeper operator policies] for details.
 
+* Enable the `auditEventsInvolvedNamespace` to manage which namespace audit event to create, and the `admissionEventsInvolvedNamespace` to to manage which namespace admission event to create. See link:../governance/create_gatekeeper.adoc#managing-gatekeeper-operator-policies[Managing Gatekeeper operator policies].
```
Comment (yiraeChristineKim) [inline release_notes/whats_new.adoc:65]: same thing here

File: governance/create_gatekeeper.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/create_gatekeeper.adoc]: @yiraeChristineKim so would it be best to update the `auditEventsInvolvedNamespace` value from `Enabled` to `true`? Which resource can the user update to update `audit-events-involved-namespace`?

Comment (dockerymick) [inline governance/create_gatekeeper.adoc]:
```suggestion
<1> To enable the `auditEventsInvolvedNamespace` parameter for you to manage which namespace audit event to create, set the parameter to `true`.
```

File: governance/create_gatekeeper.adoc (hunk 2 above)
Comment (dockerymick) [inline governance/create_gatekeeper.adoc]:
```suggestion
<2> To enable the `admissionEventsInvolvedNamespace` for you to manage which namespace admission event to create, set the parameter to `true`.
```

File: governance/create_gatekeeper.adoc (hunk 1 above)
Comment (yiraeChristineKim) [reply governance/create_gatekeeper.adoc]: Gatekeeper CR and auditEventsInvolvedNamespace = `Enabled` or `Disabled`  sorry To add `--audit-events-involved-namespace arg, set to true` I copied from Gatekeeper Doc. `Enable the `auditEventsInvolvedNamespace` to manage which namespace audit event to create.` is ok!! Sorry for confusing you

Comment (yiraeChristineKim) [reply governance/create_gatekeeper.adoc]: auditEventsInvolvedNamespace = Enabled will add arg to Gatekeeper-controller `deployment`

File: governance/create_gatekeeper.adoc (hunk 2 above)
Comment (yiraeChristineKim) [reply governance/create_gatekeeper.adoc]: Let's do not change Sorry. `Enable the `admissionEventsInvolvedNamespace` to manage which namespace admission event to create. `is enough

**Review by yiraeChristineKim — APPROVED** [review]

File: governance/create_gatekeeper.adoc (hunk 1 above)
Comment (dockerymick) [inline governance/create_gatekeeper.adoc]: What are your thoughts about this suggestion @yiraeChristineKim 
```suggestion
<1> Enable the `auditEventsInvolvedNamespace` parameter to manage which namespace audit event to create. As a result, the following argument is added to the Gatekeeper controller deployment, `--audit-events-involved-namespace: true`.
```

Comment (yiraeChristineKim) [reply governance/create_gatekeeper.adoc]: Wow this is so perfect I want to cry

File: governance/create_gatekeeper.adoc [hunk 4]
```diff
@@ -99,7 +101,9 @@ spec:
     other-annotation: "another test"
 ----
 +
-You can use the following values to manage your webhook operations. Use the following values for the `operations` parameter, `"CREATE"`, `"UPDATE"`, `"CONNECT"`, and `"DELETE"`.
+<1> Enable the `auditEventsInvolvedNamespace` parameter to manage which namespace audit event to create. As a result, the following argument is added to the Gatekeeper controller deployment, `--audit-events-involved-namespace: true`.
+<2> Enable the `admissionEventsInvolvedNamespace` to manage which namespace admission event to create.
```
Comment (dockerymick) [inline governance/create_gatekeeper.adoc]:
```suggestion
<2> Enable the `admissionEventsInvolvedNamespace` parameter to manage which namespace admission event to create. As a result, the following argument is added to the Gatekeeper controller deployment, `--admission-events-involved-namespace: true`.
```

**Review by yiraeChristineKim — APPROVED** [review]
Perfect

**Review by jc-berger — APPROVED** [review]
/lgtm

**Comment by dockerymick** [comment]
Thanks for your review and approval @jc-berger

**Pull request merged**